### PR TITLE
H2 upgrade

### PR DESCRIPTION
**Pull request by bkampe**
H2 Database before version 2.0.206 is prone to a remote command execution (RCE) vulnerability. This pull request updates H2 to version 2.1.210.

#49

### CODE REVIEW

File: example-scripts/bash-scripts/full-harvest-examples/1.6-examples/example-csv/vivo.model.xml [hunk 1]
```diff
@@ -100,7 +100,7 @@
 <Model>
     <Param name="type">sdb</Param>
     <Param name="dbLayout">layout2</Param>
-    <Param name="dbClass">com.mysql.jdbc.Driver</Param>
+    <Param name="dbClass">com.mysql.cj.jdbc.Driver</Param>
```
Comment (chenejac) [inline example-scripts/bash-scripts/full-harvest-examples/1.6-examples/example-csv/vivo.model.xml:103]: how this line is linked with h2 upgrade?

**Review by chenejac — CHANGES_REQUESTED** [review]
@bkampe thanks for this. I have posted just one minor comment. Please take a look. I didn't test your solution, but I suppose you did that.

File: example-scripts/bash-scripts/full-harvest-examples/1.6-examples/example-csv/vivo.model.xml (hunk 1 above)
Comment (bkampe) [reply example-scripts/bash-scripts/full-harvest-examples/1.6-examples/example-csv/vivo.model.xml:103]: Not sure if it is directly related to H2, but the harvester log said:
"Loading class `com.mysql.jdbc.Driver'. This is deprecated. The new driver class is `com.mysql.cj.jdbc.Driver'."
So I changed the line accordingly.

Comment (chenejac) [reply example-scripts/bash-scripts/full-harvest-examples/1.6-examples/example-csv/vivo.model.xml:103]: ok, let's keep that in this PR

**Review by chenejac — APPROVED** [review]

**Pull request merged**